packages/oscal-react-library/src/components/OSCALLoader.tsx

```diff
@@ -28,6 +29,7 @@ import {
 import { ReactElement } from "react-markdown/lib/react-markdown";
 import { AnchorLinkProps } from "./OSCALAnchorLinkHeader";
 
+const isDevMode = false; // This boolean is used to switch to dev mode
```

Comment:
Do we need a second `isDevMode` flag in the loader? Is there an issue with just using the one in the App? Maybe it can be placed in a context or be passed down as `props` to the loader.